jz/envs/README.md

```diff
@@ -160,13 +160,31 @@ pip install -e .[dev]
 cd $six_ALL_CCFRWORK/code/Megatron-DeepSpeed
 pip install -r requirements.txt
 
+# if needed first:
+# git clone https://github.com/huggingface/tokenizers $six_ALL_CCFRWORK/code/tokenizers
+cd $six_ALL_CCFRWORK/code/tokenizers
+git checkout bigscience_fork
+module load rust
+pip install setuptools_rust
+pip install -e bindings/python
```

Comment:
sorry, I was trying to say that the `~/.cargo/config.toml` workaround is needed before this line and not somewhere else. 

apologies for not being clear in the first place. would it be easier to just merge this and I will rewrite it?

Reply:
Yeah feel free to modify directly this branch.